packages/enzyme-adapter-react-16/package.json

```diff
@@ -33,7 +33,7 @@
     "enzyme-adapter-utils": "2.9.1",
     "lodash": "^4.17.4",
     "object.values": "^1.0.4",
-    "prop-types": "^15.5.10"
+    "prop-types": "^16.0.0-0"
```

Comment:
yeah. just figured this out :) but i'm getting a prop-types related failure in travis for react 16 only and i can't figure out why :/